source-code/app/src/main/java/org/buildmlearn/toolkit/activity/FirstRunActivity.java

```diff
@@ -44,11 +45,37 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         setContentView(R.layout.activity_first_run);
 
-
+        next = (Button) findViewById(R.id.button_next);
         findViewById(R.id.focus_thief).clearFocus();
-        Animation animBounceinup=AnimationUtils.loadAnimation(getBaseContext(),R.anim.bounceinup);
+        Animation anim_bounceinup=AnimationUtils.loadAnimation(getBaseContext(),R.anim.bounceinup);
```

Comment:
@DravitLochan Why using `Snake case` here? `Java` variables should be named with `lowerCamelCase` .